### PR TITLE
cloud backup / restore 

### LINKED ISSUE #127: Cloud backup and restore
Although #124 is closed, the backup and restore functions weren't described.

### LINKED ISSUE #308: document Cloud scheduled backup

### PR DESCRIPTION
**Pull request by ylorph**

### CODE REVIEW

File: docs/cloud/ops/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+The schedule format used by the Event Store Cloud CLI tool and API is a simplified subset of [cron](https://en.wikipedia.org/wiki/Cron#Overview), in the future we may support more cron features.
+
+The supported subset is:
+
+* for the first field, minute:
+    * a wildcard `*`
+    * a number between `0` and `59` (inclusive). 
+    * a rate, written as a wildcard divided by a number. E.g:  `*/15`
+* fro the second field, hour:
```
Comment (alexeyzimarev) [inline docs/cloud/ops/README.md]: For

File: docs/cloud/ops/manual_backups.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Manual Backup
+
+Backups can be created on demand through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
+
+## Console 
+
+To create a backup in the console, navigate to the clusters view and click on the create backup icon. Click `Create one off backup`.
```
Comment (alexeyzimarev) [inline docs/cloud/ops/manual_backups.md]: Then, click on `Create one-off backup`

File: docs/cloud/ops/manual_backups.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+::: card
+![take backup](./images/take_backup.png)
+:::
+
+Backups created this way appear in the console alongside backups created by scheduled jobs.
```
Comment (alexeyzimarev) [inline docs/cloud/ops/manual_backups.md]: Manually created backups..

File: docs/cloud/ops/manual_backups.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+To see the status of the backup navigate to the `Backups` section of the console. There you can see all backups created manually or by a scheduled job.
```
Comment (alexeyzimarev) [inline docs/cloud/ops/manual_backups.md]: of the backup, navigate

File: docs/cloud/ops/manual_backups.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Manual Backup
+
+Backups can be created on demand through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
```
Comment (alexeyzimarev) [inline docs/cloud/ops/manual_backups.md]: Point missing.

File: docs/cloud/ops/manual_backups.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+
+## Console
```
Comment (alexeyzimarev) [inline docs/cloud/ops/manual_backups.md]: Using the Cloud Console

File: docs/cloud/ops/manual_backups.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+:::
+
+Backups created this way appear in the console alongside backups created by scheduled jobs.
+To see the status of the backup navigate to the `Backups` section of the console. There you can see all backups created manually or by a scheduled job.
+
+::: card
+![one off backup](./images/one_off_backup.png)
+:::
+
+## ESC
```
Comment (alexeyzimarev) [inline docs/cloud/ops/manual_backups.md]: Using the command line

File: docs/cloud/ops/manual_backups.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+To create a backup using the [Event Store Cloud CLI](https://github.com/EventStore/esc):
+
+``` bash
+esc mesdb backups create --description "on demand backup" --source-cluster-id c1eut65o0aeu6ojco7a0 --project-id btfjev2rh41scaatc1k0
+BackupId("c1ev3l5o0aeu6ojco7b0")
+```
+
+To see the status of the backup use the following command:
+``` bash
+> esc  mesdb backups get  --project-id btfjev2rh41scaatc1k0 --id c1ev3l5o0aeu6ojco7b0
```
Comment (alexeyzimarev) [inline docs/cloud/ops/manual_backups.md]: Too many spaces

File: docs/cloud/ops/restore.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
+# Restore
```
Comment (alexeyzimarev) [inline docs/cloud/ops/restore.md:1]: Is there a way to use the CLI tool? If not, we need to mention this.

File: docs/cloud/ops/restore.md [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+
+To restore a backup, navigate to the `Backups` section fo the [Cloud Console](https://console.eventstore.cloud/) and click on the `Restore` icon of the backup you want to restore.
+
+::: card
+![one off restore backup](./images/one_off_restore.png)
+:::
+
+You will be then redirected to the usual provisioning page where you can choose your cluster topology
```
Comment (alexeyzimarev) [inline docs/cloud/ops/restore.md]: provisioning page, where you

Comment (alexeyzimarev) [reply docs/cloud/ops/restore.md]: topology:

File: docs/cloud/ops/scheduled_backups.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
+# Scheduled Backups
+
+Scheduled backups can be created through the [Cloud Console](https://console.eventstore.cloud/) and the [Event Store Cloud CLI](https://github.com/EventStore/esc)
+
+Scheduled backup jobs can be run as frequently as once an hour. After each successful backup, older backups created by the same job will be automatically deleted based on the provided configuration.
+
+::: note
+Multiple scheduled backups can target the same cluster, though if schedules overlap one of the jobs will fail as the cluster will already be locked.
```
Comment (alexeyzimarev) [inline docs/cloud/ops/scheduled_backups.md]: Multiple scheduled backups can target the same cluster. However, if schedules overlap, one of the jobs will fail as the first backup job will already lock the cluster.

File: docs/cloud/ops/scheduled_backups.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+:::
+For example, you could create one scheduled backup that executes every hour, along with a second scheduled backup that executes once a week. Backups from these scheduled jobs are pruned independently regardless of their age, so if both saved a maximum of four backups, the oldest backup from the weekly job might be close to a month old, while the hourly job's backups would never be older than a fraction of a day.
+
+
+## Console
```
Comment (alexeyzimarev) [inline docs/cloud/ops/scheduled_backups.md]: Using the Cloud Console

File: docs/cloud/ops/scheduled_backups.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+::: card
+![backup jobs](./images/jobs_scheduled_backup.png)
+:::
+
+There you can see all backups created by a job, as well as their history, which operations have failed (if any).
+::: note
+A backup might fail, for instance, if a cluster is locked by another operation when the backup tries to run.
+For instance if the cluster is undergoing an expand disk operation or being backed up manually.
+:::
+## ESC
```
Comment (alexeyzimarev) [inline docs/cloud/ops/scheduled_backups.md]: Using the command line

**Review by alexeyzimarev — CHANGES_REQUESTED** [review]

File: docs/cloud/ops/restore.md (hunk 9 above)
Comment (oskardudycz) [reply docs/cloud/ops/restore.md:1]: As I checked, it's possible by providing `backup_id` while creating the cluster. I created a separate issue for that https://github.com/EventStore/documentation/issues/319 to not block the whole PR.

File: docs/cloud/ops/scheduled_backups.md (hunk 13 above)
Comment (oskardudycz) [reply docs/cloud/ops/scheduled_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/scheduled_backups.md (hunk 12 above)
Comment (oskardudycz) [reply docs/cloud/ops/scheduled_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/scheduled_backups.md (hunk 11 above)
Comment (oskardudycz) [reply docs/cloud/ops/scheduled_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/restore.md (hunk 10 above)
Comment (oskardudycz) [reply docs/cloud/ops/restore.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/manual_backups.md (hunk 8 above)
Comment (oskardudycz) [reply docs/cloud/ops/manual_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/manual_backups.md (hunk 7 above)
Comment (oskardudycz) [reply docs/cloud/ops/manual_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/manual_backups.md (hunk 6 above)
Comment (oskardudycz) [reply docs/cloud/ops/manual_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/manual_backups.md (hunk 4 above)
Comment (oskardudycz) [reply docs/cloud/ops/manual_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/manual_backups.md (hunk 3 above)
Comment (oskardudycz) [reply docs/cloud/ops/manual_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/manual_backups.md (hunk 2 above)
Comment (oskardudycz) [reply docs/cloud/ops/manual_backups.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

File: docs/cloud/ops/README.md (hunk 1 above)
Comment (oskardudycz) [reply docs/cloud/ops/README.md]: Fixed in https://github.com/EventStore/documentation/pull/309/commits/a60d8e8d917d04f339f43d47322231763e0d19dd.

**Comment by oskardudycz** [comment]
@ylorph I also added documentation for the backup label options and moved the backup list to the top (from the Cloud Console description), as a backup from CLI should also be visible there.

**Review by oskardudycz — APPROVED** [review]

**Review by alexeyzimarev — APPROVED** [review]

**Pull request merged**